pkg/apis/extensions/types.go

```diff
@@ -920,6 +920,8 @@ type PodSecurityPolicySpec struct {
 	SELinux SELinuxStrategyOptions
 	// RunAsUser is the strategy that will dictate the allowable RunAsUser values that may be set.
 	RunAsUser RunAsUserStrategyOptions
+	// RunAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
+	RunAsGroup RunAsGroupStrategyOptions
```

Comment:
what's the default? (seems like it would have to be RunAsGroupStrategyRunAsAny for compatibility?)

Reply:
yes thats my understanding.